app/src/main/java/ltd/evilcorp/atox/activity/AddContactActivity.kt

```diff
@@ -2,7 +2,7 @@ package ltd.evilcorp.atox.activity
 
 import android.os.Bundle
 import android.os.Message
-import android.support.v7.app.AppCompatActivity
+import androidx.appcompat.app.AppCompatActivity
```

Comment:
Can you run the reformat code tool thing on the files you changed so the imports are sorted properly? (ctrl+alt+shift+L in Android Studio to get a dialog with some settings)

Reply:
done